hbase_sql/src/main/java/com/splicemachine/mrio/api/hive/SMHiveRecordWriter.java

```diff
@@ -87,6 +87,13 @@ public void write(RowLocationWritable key, ExecRowWritable value)
             insertOperation.open();
             insertOperation.close();
         } catch (StandardException e) {
+            if (insertOperation != null) {
+                try {
+                    insertOperation.close();
+                } catch (StandardException ex) {
+                    LOG.error(e);
+                }
+            }
```

Comment:
It looks like `insertOperation.close()` should be in a `finally` block. If line 88 throws, line 92 would throw again (or some other behavior).

Reply:
Done